### PR TITLE
Add workflow to auto-approve latest and minimum dependency pull requests

### PR DESCRIPTION
**Pull request by gsheni**
- Enabled auto-merge for minimum and latest dependency pull requests
- Created workflow to auto-approve latest and minimum dependency pull requests
- Use correct pull request GitHub Action (rather than our own fork, which we update from time to time).

### CODE REVIEW

File: .github/workflows/latest_dependency_checker.yml [hunk 1]
```diff
@@ -20,8 +20,9 @@ jobs:
         python -m pip install -e . 
         python -m pip install -r test-requirements.txt
         make checkdeps OUTPUT_PATH=featuretools/tests/requirement_files/latest_requirements.txt
-    - name: Create Pull Request
-      uses: FeatureLabs/create-pull-request@v3
```
Comment (thehomebrewnerd) [inline .github/workflows/latest_dependency_checker.yml:24]: I think we had originally used our fork of this action with the idea that changes outside our control wouldn't break our checker. Not sure if this is much of a risk or not?

**Review by thehomebrewnerd — DISMISSED** [review]

File: .github/workflows/latest_dependency_checker.yml (hunk 1 above)
Comment (gsheni) [reply .github/workflows/latest_dependency_checker.yml:24]: We've had that worry for a while and nothing has happened. If it breaks, it breaks (which I think is highly unlikely).

**Review by thehomebrewnerd — APPROVED** [review]

**Comment by tamargrey** [comment]
Nitpick - The file name `auto_approve_dependency_MRs.yml ` should probably have `PR` for github

**Comment by gsheni** [comment]
@tamargrey renamed the file.

File: .github/workflows/auto_approve_dependency_PRs.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Auto Approve Dependency MRs
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Find dependency MRs
```
Comment (tamargrey) [inline .github/workflows/auto_approve_dependency_PRs.yml:10]: There's also a bunch of `MR` usage in this file that should also get updated to PR. Sorry, forgot to mention that before!

Comment (gsheni) [reply .github/workflows/auto_approve_dependency_PRs.yml]: Good catch. Fixed.

**Review by tamargrey — APPROVED** [review]
lgtm!

**Pull request merged**